content/pl_docker/docker_course_dev.md

```diff
@@ -173,3 +173,51 @@ When developing locally on PrairieLearn, you **must** click "Load from Disk" eve
 
 That's it!
 You're ready to develop questions for the OPB!
+
+# Setting up local autograder container
+For development of questions that require an autograder, you will need to set up a local autograder container as well. 
+
+## Important! (Windows Only)
+If you are using Windows, the following commands will need to be executed in Windows Subsystem for Linux (WSL). You can do this two ways:
+1. Open a command prompt and type `wsl` to open a WSL terminal
+1. Look for the Windows Subsystem for Linux application in the Start Menu
+<img src="pl_images/wsl_icon_windows.png">
+
+## Step 1: Pull the PrairieLearn Autograder container
+
+Run the following command to pull the PrairieLearn Autograder image:
+
+```bash
+docker pull prairielearn/grader-python
```

Comment:
We could. Though I actually found that if the container isn't present, PL auto pulls the container. So step 1 technically isn't even necessary for the autograder to run. As long as the Docker command is properly configured, it should work.

Reply:
Maybe its better to just exclude the step then if prairielearn pulls the containers, and just make a note somewhere that first runs may take longer since prairielearn/docker will pull the needed docker containers if they don't exist yet - keeps the guide simpler that way I think

Reply:
I was thinking that as I was making the document. I made some changes. How does this look?

Reply:
Looks good to me!